source/patterns/@aws-solutions-constructs/aws-apigateway-sqs/lib/index.ts

```diff
@@ -214,17 +248,14 @@ export class ApiGatewayToSqs extends Construct {
         apiMethod: "GET",
         apiResource: this.apiGateway.root,
         requestTemplate: readRequestTemplate,
-        contentType: "'application/x-www-form-urlencoded'"
+        additionalRequestTemplates: props.additionalReadRequestTemplates,
+        contentType: "'application/x-www-form-urlencoded'",
+        integrationResponses: props.readIntegrationResponses
       });
     }
 
     // Delete
-    let deleteRequestTemplate = "Action=DeleteMessage&ReceiptHandle=$util.urlEncode($input.params('receiptHandle'))";
-
-    if (props.deleteRequestTemplate) {
-      deleteRequestTemplate = props.deleteRequestTemplate;
-    }
-
+    const deleteRequestTemplate = props.deleteRequestTemplate ?? "Action=DeleteMessage&ReceiptHandle=$util.urlEncode($input.params('receiptHandle'))";
```

Comment:
Pull default out and declare explicitly as a const so it is easy to discern/find later.